flink-ml-lib/src/test/java/org/apache/flink/ml/feature/MinMaxScalerTest.java

```diff
@@ -0,0 +1,208 @@
+        env = StreamExecutionEnvironment.getExecutionEnvironment(config);
+        env.setParallelism(4);
+        env.enableCheckpointing(100);
+        env.setRestartStrategy(RestartStrategies.noRestart());
+        tEnv = StreamTableEnvironment.create(env);
+        Schema schema = Schema.newBuilder().column("f0", DataTypes.of(DenseVector.class)).build();
+        DataStream<Row> dataStream = env.fromCollection(trainData);
+        trainDataTable = tEnv.fromDataStream(dataStream, schema).as("features");
+        DataStream<Row> predDataStream = env.fromCollection(predictRows);
+        predictDataTable = tEnv.fromDataStream(predDataStream, schema).as("features");
+    }
+
+    private static void verifyPredictionResult(Table output, String outputCol, DenseVector expected)
+            throws Exception {
+        StreamTableEnvironment tEnv =
+                (StreamTableEnvironment) ((TableImpl) output).getTableEnvironment();
+        DataStream<DenseVector> stream =
+                tEnv.toDataStream(output)
+                        .map(
+                                (MapFunction<Row, DenseVector>)
+                                        row -> (DenseVector) row.getField(outputCol));
+        List<DenseVector> result = IteratorUtils.toList(stream.executeAndCollect());
+        assertEquals(1, result.size());
+        assertEquals(expected, result.get(0));
+    }
+
+    @Test
+    public void testParam() {
+        MinMaxScaler minMaxScaler = new MinMaxScaler();
+        assertEquals("features", minMaxScaler.getFeaturesCol());
+        assertEquals(1.0, minMaxScaler.getMax(), 0.0001);
+        assertEquals(0.0, minMaxScaler.getMin(), 0.0001);
+        assertEquals("prediction", minMaxScaler.getPredictionCol());
+        minMaxScaler
+                .setFeaturesCol("test_features")
+                .setMax(4.0)
+                .setMin(1.0)
+                .setPredictionCol("test_output");
+        assertEquals("test_features", minMaxScaler.getFeaturesCol());
+        assertEquals(1.0, minMaxScaler.getMin(), 0.0001);
+        assertEquals(4.0, minMaxScaler.getMax(), 0.0001);
+        assertEquals("test_output", minMaxScaler.getPredictionCol());
+    }
+
+    @Test
+    public void testFeaturePredictionParam() {
+        MinMaxScaler minMaxScaler =
+                new MinMaxScaler()
+                        .setMin(1.0)
+                        .setMax(4.0)
+                        .setFeaturesCol("test_features")
+                        .setPredictionCol("test_output");
+        MinMaxScalerModel model = minMaxScaler.fit(trainDataTable.as("test_features"));
+        Table output = model.transform(predictDataTable.as("test_features"))[0];
+        assertEquals(
+                Arrays.asList("test_features", "test_output"),
+                output.getResolvedSchema().getColumnNames());
+    }
+
+    @Test
+    public void testFewerDistinctPointsThanCluster() throws Exception {
```

Comment:
Maybe we can refine this naming, as there is no concept of "cluster" in MinMaxScaler.

Reply:
Kmeans and knn user this name already, I just take it from other ut.

Reply:
KMeans is a clustering algorithm, so I understand that there is the concept of cluster. But in MinMaxScaler there is no such "cluster" thing. I think it would be more proper if we rename it to `testMinMaxEquals()`.

Reply:
OK，I got it. I will refine it later.